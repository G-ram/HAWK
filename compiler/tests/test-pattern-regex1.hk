BEGIN{}
[/c/]{print("Found c");}
END{}
