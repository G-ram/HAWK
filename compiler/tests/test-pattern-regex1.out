Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
Found c
